File: android_project/MobileCppMemoryTestAndroid/app/src/main/java/com/company/mobilecppmemorytestandroid/LoggerImpl.java
package com.company.mobilecppmemorytestandroid;

import android.util.Log;

import com.company.mobilecppmemorytest.Logger;

/**
 * Created by bruno.coelho on 27/10/2016.
 */

public class LoggerImpl extends Logger {

    @Override
    public void log(String stringLog) {
        Log.d("LoggerImpl", "logger -> " + stringLog);
    }
}

File: ios_project/MobileCppMemoryTestiOS/MobileCppMemoryTestiOS/Logger/LoggerImpl.swift
//
//  LoggerImpl.swift
//  MobileCppMemoryTestiOS
//
//  Created by Bruno Coelho on 27/10/2016.
//  Copyright © 2016 Company. All rights reserved.
//

import Foundation

class LoggerImpl: NSObject, MCMTLogger {
    
    func log(_ stringLog: String) {
        print("logger -> \(stringLog)")
    }
    
}
